File-Formats/Presentation/Working-with-find-and-replace.md

```diff
@@ -299,6 +299,79 @@ End Using
 
 You can download a complete working sample from [GitHub](https://github.com/SyncfusionExamples/PowerPoint-Examples/tree/master/Find-and-Replace/Find-first-occurance).
 
+## Find and replace using Regex pattern 
+
+You can find either the first occurrence or all the occurrences of a pattern of text in a PowerPoint presentation using the `Find` or `FindAll` method and replace it with other text.
+
+The following code example illustrates how to find all the occurrences of a pattern of text using Regex pattern and replace it with other text.
```

Comment:
The following code example illustrates how to find all the occurrences of a pattern of text using the Regex pattern and replace it with other text.